src/Template/Element/Form/date_ranges.twig

```diff
@@ -0,0 +1,15 @@
+<date-ranges-list inline-template data-items={{ dateRanges|json_encode }}>
```

Comment:
Since this is not part of a form we may wanto to use another path like `Element/Index/date_ranges.twig`